README.md

```diff
@@ -16,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-Will be delivered in the following steps of the project.
+Click here to see the live demo.
```

Comment:
I would suggest you kindly add the link to the live demo. An empty hyperlink may confuse a reader on your Readme file